packages/toast/api.js

```diff
@@ -1,12 +1,31 @@
 import { toaster as c } from '@fabric-ds/component-classes';
 import { collapse } from 'element-collapse';
-import { FabricWebComponent, windowExists } from '../utils';
+
+const windowExists = typeof window !== 'undefined';
```

Comment:
What's the rationale for bringing this in here instead of just importing it from `utils`?

Reply:
Utils has other client-side code, so the other solution here would be to move the windowExists function into its own independent file and import it wherever its used.